Source/Dafny/Verifier/Translator.ClassMembers.cs

```diff
@@ -714,10 +714,8 @@ public partial class Translator {
       if (EmitImplementation(m.Attributes)) {
         // emit impl only when there are proof obligations.
         QKeyValue kv = etran.TrAttributes(m.Attributes, null);
-        Boogie.Implementation impl = new Boogie.Implementation(m.tok, proc.Name,
-          new List<Boogie.TypeVariable>(), inParams, outParams,
-          localVariables, stmts, kv);
-        sink.AddTopLevelDeclaration(impl);
+        Boogie.Implementation impl = AddImplementationWithVerboseName(m.tok, proc,
```

Comment:
Why is it called `VerboseName`? (it's the name that's displayed to the user, right?  I think of "verbose" as the thing that has all the information but isn't very readable)

Reply:
I originally called it `DisplayName` but changed it after [this conversation](https://github.com/boogie-org/boogie/pull/586#discussion_r898182589). I'm not especially attached to any particular name.

Reply:
OK, I don't have strong feelings either.

Reply:
The name we came up with after the Boogie PR and liked most was `Description`

Reply:
And I'd be open to a future PR changing it to that. We'd have to change it in Boogie first, though.